flow/connectors/clickhouse/normalize.go

```diff
@@ -81,21 +81,23 @@ func generateCreateTableSQLForNormalizedTable(
 			return "", fmt.Errorf("error while converting column type to clickhouse type: %w", err)
 		}
 
-		if colType == qvalue.QValueKindNumeric {
+		switch colType {
+		case qvalue.QValueKindNumeric:
 			precision, scale := numeric.ParseNumericTypmod(column.TypeModifier)
 			if column.TypeModifier == -1 || precision > 76 || scale > precision {
 				precision = numeric.PeerDBClickhousePrecision
 				scale = numeric.PeerDBClickhouseScale
 			}
 			stmtBuilder.WriteString(fmt.Sprintf("`%s` DECIMAL(%d, %d), ",
 				colName, precision, scale))
-		} else {
+		case qvalue.QValueKindTimestamp, qvalue.QValueKindTimestampTZ, qvalue.QValueKindDate:
```

Comment:
Can we track nullable in QValue?

Reply:
That, and the issue specifically with clickhouse is that [Nullable affects performance and storage](https://clickhouse.com/docs/en/cloud/bestpractices/avoid-nullable-columns) and so even once we wire up nullable in qvalue, clients may not necessarily want columns to be Nullable

Reply:
yes, but that should be an option, ideally per-column

in this case it'd only be used for timestamps to be nullable when they're nullable